protocol/package.json

```diff
@@ -17,7 +17,7 @@
   },
   "homepage": "https://github.com/keybase/protocol",
   "dependencies": {
-    "avdl-compiler": "1.4.1",
+    "avdl-compiler": "1.4.2",
```

Comment:
i'd expect this to change all the protocol output. Did you `make clean` then `make` in `protocol`?